Woof/Woof/ViewModel/SitterProfileViewModel.swift

```diff
@@ -46,6 +50,22 @@ final class SitterProfileViewModel: ObservableObject {
             .save(data, for: KeyValueStorage.Key.currentSitter)
     }
 
+    /// Uploads the pet sitter's data to the backend server asynchronously.
+    ///
+    /// `isErrorOccurred` is set to `false`, indicating no errors.
+    /// If there is a network error during the upload,`isErrorOccurred` is set to `true`.
+    func upload() async throws {
```

Comment:
Where do we intend to call this function?

Reply:
In SitterProfileView